pkg/network/tracer/tracer_test.go

```
@@ -1529,14 +1528,17 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 	serverDoneFn := testutil.HTTPServer(t, "127.0.0.1:443", enableTLS)
 	defer serverDoneFn()
 
+	// Run wget once to make sure the OpenSSL is detected and uprobes are attached
+	exec.Command(wget).Run()
+	time.Sleep(time.Second)
+
 	// Issue request using `wget`
 	// This is necessary (as opposed to using net/http) because we want to
 	// test a HTTP client linked to OpenSSL
 	const targetURL = "https://127.0.0.1:443/200/foobar"
 	requestCmd := exec.Command(wget, "--no-check-certificate", "-O/dev/null", targetURL)
-	err1 := requestCmd.Start()
-	err2 := requestCmd.Wait()
-	if err1 != nil || err2 != nil {
+	err = requestCmd.Run()
+	if err != nil {
 		t.Skip("failed to issue request command; skipping test.")
```

Should this skip the test, or fail it?

---

yeah, better fail loudly in this case 👍